OpenTap.Plugins.PNAX/BaseSteps/SingleTraceBaseStep.cs

```diff
@@ -172,7 +175,7 @@ public virtual void UpdateTestStepName()
             }
 
             Trace = $"CH{Channel}_{finalMeasEnumName}";
-            Name = $"CH{Channel}_{finalMeasEnumName}";
+            Name = "CH{Channel}_{finalMeasEnumName}";
```

Comment:
Do you really want to remove the $ here? You'll end up with a string like this instead?
CH{Channel}_{finalMeasEnumName}